packages/waterfall/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@mui-internal/waterfall",
```

Comment:
```suggestion
  "name": "@mui/internal-waterfall",
```
Would this be better? To have the same org name

Reply:
Sure, I'm OK with both.

Reply:
There are other packages that use the @mui-internal scope. I'll rename them in a separate PR.